.travis.yml

```diff
@@ -1,20 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
I know this isn't the complete file but can you base what you have got on the latest structure in the `template-formula`?